src/force/CabanaPD_Force_LPS.hpp

```diff
@@ -76,6 +76,9 @@ class Force<ExecutionSpace, ForceModel<LPS, Elastic>>
     bool _half_neigh;
     ForceModel<LPS, Elastic> _model;
 
+    Timer _timer;
```

Comment:
Should we use "_force_timer" (instead of "_timer") for consistency with "_energy_timer;"?

Reply:
I've tried to be consistent about the "main" timer being called `timer` in every class